uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1280,7 +1280,9 @@ sub compute_hash {
 	$ctx->add($file->getParameter('patient:birthdate'));          # Patient DOB
 	$ctx->add($file->getParameter('study_instance_uid'));         # StudyInstanceUID
 	$ctx->add($file->getParameter('series_description'));         # SeriesDescription
-    $ctx->add($file->getParameter('processing:intergradient_rejected')); 
+    if ($file->getParameter('processing:intergradient_rejected')) {
```

Comment:
I do not know if this is possible but if parameter `processing:intergradient_rejected` exists and is either `''` or `0`, then this tests evaluates to false. Is this what you want?

Reply:
good catch!!! Should be corrected in the next commit.